### DISCUSSION TITLE
BBT cannot export references of type "standard".

### DISCUSSION THREAD

**Discussion by MysticDragonfly** [retorquere/zotero-better-bibtex#2542] [Q&A]
I won't file a bug, but I hope to share the solution to the problem with you in this way.
BBT cannot export references of type "standard". I opened the ../workder/zotero.js file, and after searching and comparing, I found that the "standard" item was missing in some places. Specifically: 128764 lines, 131569 lines. Documents of type "standard" will be marked as "msic" if not modified. After supplementing these two omissions, the documents marked with "standard" can be exported correctly.
corrected code：
**line 128764 ：**     
 types: /* @__PURE__ */ new Set(["article", "book", "bookinbook", "booklet", "collection", "inbook", "incollection", "inproceedings", "inreference", "manual", "misc", "mvbook", "mvcollection", "online", "patent", "periodical", "proceedings", "reference", "report", "set", "suppbook", "suppcollection", "suppperiodical", "thesis", "unpublished", "standard"]),
**lines 131568-131570**：
        report: "techreport",
        standard: "standard",
        statute: "misc",

**Comment by retorquere** [comment]
I'd much prefer it if you would file a bug.
